server/src/main/java/com/objectcomputing/checkins/services/guild/member/GuildMemberController.java

```diff
@@ -83,8 +113,19 @@ public GuildMemberController(GuildMemberServices guildMemberServices) {
      * @return {@link GuildMember}
      */
     @Get("/{id}")
-    public GuildMember readGuildMember(UUID id) {
-        return guildMemberServices.read(id);
+    public Single<HttpResponse<GuildMember>> readAgendaItem(UUID id) {
+        return Single.fromCallable(() -> {
+            GuildMember result = guildMemberServices.read(id);
```

Comment:
Although not necessary it would be cleaner and more consistent with other services if this null check and exception handling is done in the service impl. Try using `orElseThrow(()->new GuildMemberNotFoundExceptio())`